packages/components/buttons/flat-button/src/flat-button.tsx

```diff
@@ -8,15 +8,15 @@ import {
 } from 'react';
 import { css } from '@emotion/react';
 import omit from 'lodash/omit';
-import { customProperties } from '@commercetools-uikit/design-system';
+import { designTokens } from '@commercetools-uikit/design-system';
 import { filterInvalidAttributes } from '@commercetools-uikit/utils';
 import Text from '@commercetools-uikit/text';
 import AccessibleButton from '@commercetools-uikit/accessible-button';
 import { getTextColor, getButtonIconColor } from './flat-button.styles';
 
 const propsToOmit = ['type'];
 
-export type CustomProperties = typeof customProperties;
+export type TDesignTokens = typeof designTokens;
```

Comment:
I renamed this type for consistency.
Although it is exported, it is only used in the `flat-button.styles.ts` file and no exported from the package, so it's safe.